homeassistant/components/accuweather/sensor.py

```diff
@@ -156,37 +149,32 @@ class AccuWeatherSensorDescription(SensorEntityDescription):
     ),
     AccuWeatherSensorDescription(
         key="SolarIrradianceDay",
-        icon="mdi:weather-sunny",
         entity_registry_enabled_default=False,
         native_unit_of_measurement=UnitOfIrradiance.WATTS_PER_SQUARE_METER,
         value_fn=lambda data: cast(float, data[ATTR_VALUE]),
         translation_key="solar_irradiance_day",
     ),
     AccuWeatherSensorDescription(
         key="SolarIrradianceNight",
-        icon="mdi:weather-sunny",
         entity_registry_enabled_default=False,
         native_unit_of_measurement=UnitOfIrradiance.WATTS_PER_SQUARE_METER,
         value_fn=lambda data: cast(float, data[ATTR_VALUE]),
         translation_key="solar_irradiance_night",
     ),
```

Comment:
There is a device class for irradiance. I think we can use that device class and use the icons provided by the device calss

Reply:
Didn't know, I'll update the PR, thanks.